syntax = "proto3";

package tensorflow_federated.v0;

import "google/protobuf/any.proto";
import "tensorflow_federated/proto/v0/computation.proto";

message IterativeProcess {
  Computation initialize_fn = 1;
  Computation next_fn = 2;
}
